PHPCompatibility/Tests/Classes/NewClassesUsesUnitTest.inc

```diff
@@ -0,0 +1,377 @@
+<?php
```

Comment:
The sniff (correctly) stores and checks the import statements per file.

However, the test as-is does not test/safeguard that there is no leakage of import statements between files. This means that if the "storing per file" is broken at some point, that would not be flagged by the tests.

Could this be tested in some way ?

Reply:
I modified the test which should take care of that now.

Reply:
Thank you for that and the test looks good. Well done making sure that the same sniff instance is being used for both files. 🙌🏻